platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/communication/handshake/VersionCompareHandshake.java

```diff
@@ -59,6 +59,9 @@ public VersionCompareHandshake(final SoftwareVersion version, final boolean thro
     @Override
     public void runProtocol(final Connection connection)
             throws NetworkProtocolException, IOException, InterruptedException {
+        logger.info(
```

Comment:
remember to remove this, i added it for debugging

Reply:
removed